golemcli/src/main.rs

```diff
@@ -52,6 +52,12 @@ struct CliArgs {
     #[structopt(long, short)]
     net: Option<Net>,
 
+    /// Answer `YES` to any prompt without an user action
+    #[structopt(short = "y")]
+    #[structopt(raw(display_order = "500"))]
```

Comment:
I think it will be rather not popular so I'd sort it last
```suggestion
    #[structopt(raw(display_order = "1000"))]
```

Reply:
ok